src/components/settings-table/program-table-row.js

```diff
@@ -1,26 +1,18 @@
 import React from 'react'
-
-import Select from '@material-ui/core/Select'
-import MenuItem from '@material-ui/core/MenuItem'
-import TableRow from '@material-ui/core/TableRow'
-import TableCell from '@material-ui/core/TableCell'
-import InputNumber from '../input-number'
 import i18n from '@dhis2/d2-i18n'
-import dataTableStyles from '../../styles/DataTable.module.css'
 import tableTitleStyles from '../../styles/TableTitle.module.css'
 import {
     specificSettingsDefault,
     TEI_DOWNLOAD,
 } from '../../constants/program-settings'
+import { InputNumber, SelectSettings } from '../inputs'
+import TableRow from './table-row'
+import { Divider } from '@dhis2/ui'
 
-const ProgramTableRow = ({ dataRow, states, onChange }) => {
-    return (
+const ProgramTableRow = ({ dataRow, states, onChange }) => (
+    <div>
         <TableRow>
-            <TableCell
-                component="th"
-                scope="row"
-                className={dataTableStyles.dataTable__rows__row__column}
-            >
+            <div>
```

Comment:
Why is this no longer a `TableCell`?

Reply:
This is part of Joe's suggestions [(sketch)](https://www.sketch.com/s/7437a41c-0ddc-44ea-bfac-d75ca6620bae/a/7QjjOY/play)